samples/cpp/speech_sample/speech_sample.hpp

```diff
@@ -228,8 +222,6 @@ static void show_usage() {
     std::cout << "    -we \"<path>\"               " << write_embedded_model_message << std::endl;
     std::cout << "    -cw_l \"<integer>\"          " << context_window_message_l << std::endl;
     std::cout << "    -cw_r \"<integer>\"          " << context_window_message_r << std::endl;
-    std::cout << "    -oname \"<string>\"          " << output_layer_names_message << std::endl;
-    std::cout << "    -iname \"<string>\"          " << input_layer_names_message << std::endl;
```

Comment:
message input_message, output_message and reference_score_message  should be updated too